superset-frontend/src/views/CRUD/data/dataset/AddDataset/index.tsx

```diff
@@ -106,6 +107,14 @@ export default function AddDataset() {
       getDatasetsList();
     }
   }, [dataset?.schema]);
+  const [showEditPage, setShowEditPage] = useState(false);
```

Comment:
another wording suggestion: how about something like `editPageIsVisible` and `setEditPageIsVisible`? Just so it sounds like a boolean. You _could_ maybe leave out `is` if you wanted, too.

Reply:
I like that change, set to `editPageIsVisible` in [`this commit`](https://github.com/apache/superset/pull/22043/commits/77f5c7f40fe0c0269d5ef8a2f3e6ad2483d8e180).